tests/test_view/test_views.py

```diff
@@ -231,33 +43,34 @@ def test_rate(client: TestClient):
     assert client.get("/cryptos/rate?fiat_currency=test").status_code == 422
 
 
-def test_wallet_history(client: TestClient, token: str):
+def test_wallet_history(client: TestClient, token: str, wallet):
     headers = {"Authorization": f"Bearer {token}"}
-    assert client.get("/wallets/history/1", headers=headers).status_code == 404
```

Comment:
This test now doesn't test non-existing wallets and other edge cases. Is it intentional?